weave/wandb_interface/wandb_artifact_pusher.py

```diff
@@ -92,8 +97,8 @@ def write_artifact_to_wandb(
 
     # Return the URI of the artifact
     return WeaveWBArtifactURIComponents(
-        entity_name=lite_run.run.entity,
```

Comment:
This actually triggers a duplicate cascade of UpsertModel, UpsertBucket, etc. because the `lite_run.finish()` call above sets _run in the InMemoryLazyLiteRun class to None.

Reply:
nice find